certbot/tests/main_test.py

```diff
@@ -1059,23 +1059,6 @@ def test_certonly_renewal(self, unused_notafter):
         self.assertTrue('fullchain.pem' in cert_msg)
         self.assertTrue('donate' in get_utility().add_message.call_args[0][0])
 
-    # Should be moved to renewal_test.py
-    @test_util.broken_on_windows
-    @mock.patch('certbot.crypto_util.notAfter')
-    def test_certonly_renewal_triggers(self, unused_notafter):
-        # --dry-run should force renewal
-        _, get_utility, _ = self._test_renewal_common(False, ['--dry-run', '--keep'],
-                                                      log_out="simulating renewal")
-        self.assertEqual(get_utility().add_message.call_count, 1)
-        self.assertTrue('dry run' in get_utility().add_message.call_args[0][0])
-
-        self._test_renewal_common(False, ['--renew-by-default', '-tvv', '--debug'],
-                                  log_out="Auto-renewal forced")
```

Comment:
Is the call with log_out="Auto-renewal forced" not necessary for the tests? If not maybe we should add a comment on why it isn't needed.

Reply:
Good catch. It's probably for the best to keep it.